lib/backends/cpu/ops/argMax.ts

```diff
@@ -10,13 +10,13 @@ export class CpuArgMax extends ArgMax {
   }
 }
 
-export function argMax(x: Tensor, axis: number, keepdims: number): Tensor {
+export function argMax(x: Tensor, axis: number, keepdims: boolean): Tensor {
   const rank = x.dims ? x.dims.length : 1;
   axis = ShapeUtil.parseAxis(axis, rank);
-  const outputDims = ReduceUtil.calcReduceShape(x.dims.slice(0), [axis], 1);
+  const outputDims = ReduceUtil.calcReduceShape(x.dims, [axis], true);
   const X = x.data;
   const Y = new Int32Array(ShapeUtil.size(outputDims));
-  const blockSize = axis >= x.dims.length ? 1 : ShapeUtil.size(x.dims.slice(axis + 1));
+  const blockSize = axis + 1 >= x.dims.length ? 1 : ShapeUtil.sizeFromDimension(x.dims, axis + 1);
```

Comment:
dumb question 1: if `axis +1 > x.dims.length`, shouldn't it be an error case ?

dumb question 2: can't we directly call sizeFromDimension(x.dims, axis + 1) and it should return 1 if axis +1 == x.dims.length  ?

Reply:
good catch! I checked the code and found it can be simplified to `sizeFromDimension(x.dims, axis + 1) `